src/OpenTelemetry/Trace/Builder/TracerProviderBuilderBase.cs

```diff
@@ -41,7 +41,8 @@ public TracerProviderBuilderBase()
             .AddOpenTelemetrySharedProviderBuilderServices()
             .AddOpenTelemetryTracerProviderBuilderServices()
             .TryAddSingleton<TracerProvider>(
-                sp => throw new NotSupportedException("Self-contained TracerProvider cannot be accessed using the application IServiceProvider call Build instead."));
+                sp => throw new NotSupportedException(
+                    "Self-contained TracerProvider cannot be accessed using the application IServiceProvider call Build instead."));
```

Comment:
@CodeBlanch any suggestion how to improve the wording here? e.g. "what happened; what to do"

Reply:
I didn't even mean to touch that, I think the rider formatter just did a little too much formatting, let me revert that.